ibm_cloud_sdk_core/token_managers/iam_request_based_token_manager.py

```diff
@@ -95,7 +96,7 @@ def request_token(self) -> dict:
              A dictionary containing the bearer token to be subsequently used service requests.
         """
         headers = {
-            'Content-type': 'applicaton/x-www-form-urlencoded',
+            'Content-type': 'application/x-www-form-urlencoded',
```

Comment:
wow, the misspelled "applicaton" didn't cause a problem before???  :)

Reply:
I was surprised too! The container authenticator tests will cover this, so I didn't change the tests here.